src/devtools/views/Components/Element.js

```diff
@@ -29,6 +30,7 @@ export default function ElementView({ index, style }: Props) {
     selectedElementID,
     selectElementByID,
   } = useContext(TreeContext);
+  const lastScrolledIDRef = useContext(ScrollAnchorContext);
```

Comment:
Probably cheaper to just use `itemData` for this rather than adding a new context. Thoughts?